ddtrace/internal/products.py

```diff
@@ -192,6 +193,8 @@ def _do_products(self) -> None:
         atexit.register(self.exit_products)
 
     def run_protocol(self) -> None:
+        self._load_products()
```

Comment:
It looks like there are other public functions defined in this class, is there any risk we'd call one of those methods without loading the products?